userspace/falco/app/actions/load_rules_files.cpp

```diff
@@ -126,13 +126,13 @@ falco::app::run_result falco::app::actions::load_rules_files(falco::app::state&
 
 	if (s.options.describe_all_rules)
```

Comment:
I would initialize both `describe_all_rules` and `describe_rule` variables here

```cpp
options::options()
	: event_buffer_format(sinsp_evt::PF_NORMAL),
	  gvisor_config(""),	
	  list_fields(false),
	  list_plugins(false),
	  list_syscall_events(false),
	  markdown(false),
	  modern_bpf(false)
{
}
```

Reply:
They should already have their default I guess in options.cpp